docs-2.0-en/5.configurations-and-logs/1.configurations/4.storage-config.md

```diff
@@ -80,7 +80,7 @@ For all parameters and their current values, see [Configurations](1.configuratio
 
 | Name                             | Predefined value | Description               |Whether supports runtime dynamic modifications|
 | :------------------------------- | :--------------- | :------------------------ |:------------------|
-| `data_path`                      | `data/storage` | Specifies the data storage path. Multiple paths are separated with commas. For NebulaGraph of the community edition, one RocksDB instance corresponds to one path.|  No|
+| `data_path`                      | `data/storage` | Specifies the data storage path. Multiple paths are separated with commas. For NebulaGraph of the community edition, one RocksDB instance corresponds to one path. Do not modify this parameter abitrarily to prevent accidental data loss. |  No|
```

Comment:
```suggestion
| `data_path`                      | `data/storage` | Specifies the data storage path. Multiple paths are separated with commas. For NebulaGraph of the community edition, one RocksDB instance corresponds to one path. Do not modify this parameter arbitrarily to prevent accidental data loss. |  No|
```